hr_employee_product/security/product_security.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="0">
+        <record id="property_rule_product_employee_user" model="ir.rule">
```

Comment:
Not required, yet please consider adding a copyright and license to XML as well, since it's kind-a code

Reply:
Copyright and license added. Thanks for your review